.github/workflows/lint.yml

```
@@ -0,0 +1,22 @@
+name: lint
+on: [push]
+jobs:
+  app:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: app
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Read .nvmrc
+        run: echo ::set-output name=NVMRC::$(cat .nvmrc)
+        id: nvm
+      - name: Use Node.js ${{ steps.nvm.outputs.NVMRC }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: '${{ steps.nvm.outputs.NVMRC }}'
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
+      - name: Lint TypeScript
+        run: yarn run lint
```

```suggestion
      - name: Lint Typescript/Javascript
        run: yarn run lint
      - name Typecheck
        run: yarn run tsc
```

another check we can add

---

i think you need a `:` after name on line 23

---

I added `tsc --noEmit` to the npm `lint` script to keep things in one place. Thanks for the feedback!